### ISSUE TITLE
Hi，where is the KL loss in your code

### ISSUE THREAD

**Issue by weiaicunzai** [invoker-LL/WSI-finetuning#5]
Hi, thanks for your interesting work. I have a question about your loss function (Eq.(8)) in your paper.
In Eq.(8), you have stated that we should use KL loss in the latter part.  However, I didn't see any KL loss code in your  code (All I have found is [MSE loss](https://github.com/invoker-LL/WSI-finetuning/blob/e1c6972a3a1fa47b2e3497fb042cbc8a6372431d/models/vib_clam_wsi.py#L144C37-L144C37) ). Could you please tell me where did you use your KL loss in your code? Thanks.

**Comment by aitrics-jeremy** [comment]
Thank you for your interesting work!

I have a same question with weliaicunzai. 
I can't find the KL divergence loss in your code. Where did you implement the KL divergence loss? 

Thanks.

**Comment by invoker-LL** [comment]
Since the Bernoulli distribution is easy with given probability (p=0.5), a MSE loss is easy to implement when aiming at

**Comment by invoker-LL** [comment]
MSE and KL divergence are both metrics/loss for distribution alignment as used in Knowledge Distillation. Since Bernoulli distribution is easy with given constant probability like p=0.2, it can be seen as a soft-target for a binary classification, thus we can also use CrossEntropy to replace the KL given pre-defined distribution. I will provide this code soon.
